constraints.rst

```diff
@@ -219,7 +219,7 @@ This constraint can do two things:
 1. Constrains two lines to be parallel.
 
 :Workplane: none
-:Selection: Line (2D only) and arc
+:Selection: Two lines (2D only)
```

Comment:
3D lines also work.

Reply:
You are correct. I fixed it and force pushed my branch.